stdlib/2and3/imghdr.pyi

```diff
@@ -1,16 +1,21 @@
-from typing import overload, Union, Text, BinaryIO, Optional, Any, List, Callable
-import sys
 import os
+import sys
+from typing import Any, BinaryIO, Callable, List, Optional, Protocol, Text, Union, overload
+
+class _ReadableBinary(Protocol):
+    def tell(self) -> int: ...
+    def read(self, size: int = ...) -> bytes: ...
+    def seek(self, offset: int) -> Optional[int]: ...
```

Comment:
We don't use the return of `seek` so I annotated this as `Optional[int]` - i.e. you don't have to return anything but if you do, an `int` is the right thing to return.

Reply:
I'd recommend to use `Any`, meaning "we don't care about the return code".

Reply:
Will do.